tests/integration/commands/test_command_test.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from pytest import CaptureFixture
+
+from tests.integration.conftest import CreateProtostarProjectFixture, ProtostarFixture
+from tests.conftest import DevnetFixture
+
+
+@pytest.fixture(autouse=True, name="protostar")
+def protostar_fixture(create_protostar_project: CreateProtostarProjectFixture):
+    with create_protostar_project() as protostar:
+        protostar.build_sync()
+        yield protostar
+
+
+async def test_gas_estimation(
+    protostar: ProtostarFixture,
+    devnet: DevnetFixture,
+    capsys: CaptureFixture[str],
+):
+    await protostar.test(
+        targets=["::test_increase_balance"], gas_price=devnet.get_gas_price()
```

Comment:
We do we need to query `gas_price` from devnet? Cannot we just set it to 1?

Reply:
It's not needed, actually. Initially, I wanted to test the estimation result against `starknet estimate-fee`.